src/Shared/PasswordGenerator.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why is this in the `Shared` directory, but only used in 1 project? Can we just put it in the Aspire.Hosting folder?